src/MultiOwnerLightAccount.sol

```diff
@@ -162,6 +228,14 @@ contract MultiOwnerLightAccount is BaseLightAccount, CustomSlotInitializable {
         return false;
     }
 
+    /// @dev Convert a boolean success value to a validation data value.
+    /// @param success The success value to be converted.
+    /// @return validationData The validation data value. 0 if success is true, 1 (SIG_VALIDATION_FAILED) if
+    /// success is false.
+    function _successToValidationData(bool success) internal pure returns (uint256 validationData) {
+        return success ? SIG_VALIDATION_SUCCESS : SIG_VALIDATION_FAILED;
```

Comment:
I previously had this as just `iszero` in assembly, but I figured it might make sense to keep this simple and readable. The gas difference is about 20, so not very significant, but also OK to switch back.